### PR TITLE
[11.0] Migrated sale_last_price_info module

### PR DESCRIPTION
**Pull request by Swapnesh-SerpentCS**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis

**Comment by JayVora-SerpentCS** [comment]
I have emailed CLA request already

**Comment by JayVora-SerpentCS** [comment]
@Swapnesh-SerpentCS please squash the transbot commits.

File: sale_last_price_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_last_price_info%0Aversion:%2011.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```
Comment (JayVora-SerpentCS) [inline sale_last_price_info/README.rst:26]: @Swapnesh-SerpentCS Please update the bugtracker as per https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: sale_last_price_info/README.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (JayVora-SerpentCS) [inline sale_last_price_info/README.rst]: https

**Comment by JayVora-SerpentCS** [comment]
@Swapnesh-SerpentCS do check

File: sale_last_price_info/README.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    :alt: License: AGPL-3
+
+==============================
+Product Last Price Info - Sale
+==============================
+
+This module adds the last sale info of the product.
+
+
+Bug Tracker
```
Comment (JayVora-SerpentCS) [inline sale_last_price_info/README.rst:20]: Add runbot section

File: sale_last_price_info/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+
+{
+    "name": "Product Last Price Info - Sale",
+    "version": "11.0.1.0.0",
+    "author": "OdooMRP team, "
+              "AvanzOSC, "
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Odoo Community Association (OCA)",
+    "website": "http://www.odoomrp.com",
```
Comment (JayVora-SerpentCS) [inline sale_last_price_info/__manifest__.py]: website to be https://github.com/OCA/sale-workflow

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by Swapnesh-SerpentCS** [comment]
Updated accordingly.  @JayVora-SerpentCS

**Review by JayVora-SerpentCS — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
LGTM. @pedrobaeza

**Review by YogeshMahera-SerpentCS — APPROVED** [review]
:+1:

**Pull request merged**